dev_guide/templates.adoc

```diff
@@ -51,41 +51,22 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
 
-There is also the ability to add labels in the template from the command line.
+You can create a configuration from a template with either the Management Console or the CLI, after first uploading a template to your OpenShift instance.
 
-****
-`$ osc process -f _<filename>_ -l name=otherLabel`
-****
+Create a template json file, like the above example, then upload it with the the CLI using the following process:
 
-== Parameters
-The list of parameters that you can override are listed in the `*parameters*` section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+=== Uploading a Template
+Load a json template file into the OpenShift template library with the following command:
 
 ****
-`$ osc process --parameters -f _<filename>_`
+`osc create -f _<filename>_`
 ****
 
-The following shows the output when listing the parameters for one of the *_openshift/origin/examples_* templates.
-
-.Example Template Output
-====
-
-----
-$ osc process --parameters -f examples/sample-app/application-template-dockerbuild.json
-NAME                  DESCRIPTION              GENERATOR           VALUE
-ADMIN_USERNAME        administrator username   expression          admin[A-Z0-9]{3}
-ADMIN_PASSWORD        administrator password   expression          [a-zA-Z0-9]{8}
-MYSQL_ROOT_PASSWORD   database password        expression          [a-zA-Z0-9]{8}
-MYSQL_DATABASE        database name                                root
-----
-
-====
+The template is now available to be selected for a configuration using the Management Console or the CLI.
 
-The output identifies several parameters that are generated with a regex expression generator when the template is processed.
-
-== Generating a Configuration
+=== Generating a Configuration
 When OpenShift processes a template, it creates a configuration. You can override the parameters with the values defined in the configuration file by adding the appropriate flags.
```

Comment:
If you do move this "When OpenShift processes a template, it creates a configuration" line up, as suggested above, then the 2nd sentence here could just move to after the following example, where it would align with the example about overriding parameters.